CONTRIBUTING.md

```diff
@@ -8,9 +8,9 @@
 ├── .github/workflows         GitHub Actions configurations
 ├── main.go                   command entry
 ├── pkg
-│   ├── catalog
-│   ├── catalogdiagrams
-│   └── watcher
+│   ├── catalog               core logic to generate catalog files and serve service
+│   ├── catalogdiagrams       functions to generate diagrams
+│   └── watcher               watcher to auto-update live server
```

Comment:
I would probably phrase it as
`functions to watch for file changes in server mode`